lib/system.nim

```diff
@@ -4198,6 +4198,11 @@ type
   NimNode* {.magic: "PNimrodNode".} = ref NimNodeObj
     ## Represents a Nim AST node. Macros operate on this type.
 
+macro lenVarargs*(x: varargs[untyped]): int =
```

Comment:
Wrap it in: ``when (NimMajor, NimMinor) >= (1, 1)`` please.

Reply:
I'm assuming you meant ` {.since: (1, 1).}` (otherwise, not sure why `since` would be added); I added  `{.since: (1, 1).}`